_projects/git_task_list.md

```diff
@@ -0,0 +1,87 @@
+---
+collaborating_projects:
+  - coala
```

Comment:
no.  your project does not intend to enhance `coala/coala`.

here you would put `gsoc-prep-tasks` as the related project